app/screens/search/search.js

```diff
@@ -226,17 +224,17 @@ class Search extends Component {
     };
 
     renderPost = ({item, index}) => {
-        if (item.id === SEARCHING) {
+        const {channels, posts, theme} = this.props;
+        const style = getStyleFromTheme(theme);
+
+        if (item.id === SEARCHING || item.id === NO_RESULTS) {
             return (
-                <View style={{width: '100%', height: '100%'}}>
+                <View style={[style.customItem]}>
```

Comment:
totally forgot to remove it, I'll do that then I'll merge this since you approved